PWGUD/AQC/udQCmidRap.cxx

```diff
@@ -348,19 +362,14 @@ struct UDQCmid {
       }
     }
 
-    // 4. Check for global tracks which are no vtx tracks
+    // 4. Check for global tracks
     bool globalAndVtx = isDGcandidate;
-    bool vtxAndGlobal = isDGcandidate;
     for (auto const& track : tracks) {
-      if (track.isGlobalTrack() && !track.isPVContributor()) {
+      if (track.isGlobalTrack() && track.isPVContributor()) {
```

Comment:
Is this really what you want? I would thin that in case that

track.isGlobalTrack() && track.isPVContributor()

is true, you want to set vtxAndGlobal to true.

Reply:
Yes, the First condition is to look for PVcontributor then if the tracks are global tracks or ITS only tracks